CONTRIBUTING.md

```diff
@@ -1,57 +1,169 @@
-
-
 # Contributing to vcd-cli
 
-The *vcd-cli* project team welcomes contributions from the community. If you wish to contribute code and you have not
-signed our contributor license agreement (CLA), our bot will update the issue when you open a Pull Request. For any
-questions about the CLA process, please refer to our [FAQ](https://cla.vmware.com/faq).
+Welcome! We gladly accept contributions from the community. If you wish
+to contribute code and you have not signed our contributor license
+agreement (CLA), our bot will update the issue when you open a pull
+request. For any questions about the CLA process, please refer to our
+[FAQ](https://cla.vmware.com/faq).
 
 ## Community
 
 https://vmwarecode.slack.com, #vcd channel
 
-## Contribution Flow
+## Logging Bugs
+
+Anyone can log a bug using the Github 'New Issue' button.  Please use
```

Comment:
I believe `GitHub` is the right capitalization.

Reply:
Thanks, will fix.